Vagrantfile

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
 
   # Run Ansible from the Vagrant VM
   config.vm.provision "ansible_local" do |ansible|
-    ansible.playbook = "coldfront_app/coldfront/bootstrap/development/playbook.yml"
+    ansible.playbook = "coldfront_app/coldfront/bootstrap/ansible/playbook.yml"
     ansible.galaxy_role_file = "coldfront_app/coldfront/bootstrap/development/requirements.yml"
```

Comment:
The requirements file is accounted for when provisioning the dev. VM.

On production, in the past, I installed requirements manually as needed (e.g., `ansible-galaxy collection install community.general`). Is there a command to install them from the `.yml`, and if so, can you include it in `README.md`?